src/base-button/base-button.ts

```diff
@@ -67,7 +67,9 @@ export abstract class BaseButton extends LitElement {
   private renderLoadingContent = () => {
     return html `
       <div class="spinner">
-        <tap-spinner></tap-spinner>
+        <tap-spinner
+          size=${this.size === 'small' ? 'small' : nothing}
+        ></tap-spinner>
```

Comment:
I think both the `button` and `spinner` components have the property `size` with the type of `'small' | 'medium' | 'large'`. Can't we pass the `size` of the `button` to the `spinner`?

```suggestion
        <tap-spinner size=${this.size}></tap-spinner>
```

Reply:
No, because the sizes of the two components do not correspond one-to-one. Buttons labeled 'large' and 'medium' have a spinner with a 'medium' size.